src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
no, it can not, but it can ask for block "from the future" and we should test that it will not return their hashes even if they are in storage